src/components/settings/LocationSettings.js

```diff
@@ -32,6 +32,8 @@ const useSystemPermission = () => {
 const getLocationMarker = (locationObject) => ({
   ...baseLocationMarker,
   position: {
+    latitude: locationObject?.coords?.lat,
+    longitude: locationObject?.coords?.lng,
     ...locationObject.coords
```

Comment:
this can be removed now, right?

Reply:
I recommend it not to be removed. There may be other features inside the object.

Reply:
ok  than please change the order and put the spreading before the hardcoded keys. otherwise `latitude` and `longitude` would be overridden with values in `coords` if existing